src/app/features/request/new/new.component.html

```diff
@@ -35,8 +36,10 @@ <h1 class="font-bold text-white text-6xl pb-8">طلب عرض سعر..</h1>
                         name="request"
                         id="request"
                         autocomplete="request"
-                        class="block flex-1 w-full border border-slate-800 bg-slate-50 rounded-md py-1.5 pr-3 text-gray-900 placeholder:text-gray-400 focus:ring-0 sm:text-sm sm:leading-6"
-                        placeholder="علي سبيل المثال , ببناء موقع علي شبكة الانترنت"
+                        class="block flex-1 w-full border border-slate-800 bg-slate-50 rounded-md py-1.5 ps-3 text-gray-900 placeholder:text-gray-400 focus:ring-0 sm:text-sm sm:leading-6"
+                        placeholder="{{
+                          'translate_request_title_placeholder' | translate
+                        }}"
```

Comment:
```suggestion
                        [placeholder]="'translate_request_title_placeholder' | translate"
```

Do the same in other placehoders